Core/Core/View/Base/CourseCellView.swift

```diff
@@ -131,7 +143,7 @@ public struct CourseCellView: View {
         .opacity(showView ? 1 : 0)
         .offset(y: showView ? 0 : 20)
         .accessibilityElement(children: .ignore)
-        .accessibilityLabel(courseName + " " + (type == .dashboard ? (courseEnd == "" ? courseStart : courseEnd) : ""))
+//        .accessibilityLabel(courseName + " " + (type == .dashboard ? (courseEnd == "" ? courseStart : courseEnd) : ""))
```

Comment:
why you disabled accessibility label here?

Reply:
It was disabled while testing the feature, enabled it with correct values.